environment.yml

```diff
@@ -1,13 +1,9 @@
 name: cta
 channels:
   - default
-  - cta-observatory
-  - conda-forge
 dependencies:
-  - python=3.7
-  - ctapipe
-  - gammapy
-  - pip:
```

Comment:
Maybe add _-pip_ before listing the pip dependencies to avoid conda's _Warning: you have pip-installed dependencies in your environment file, but you do not list pip itself as one of your conda dependencies.  Conda may not use the correct pip to install your packages, and they may end up in the wrong place.  Please add an explicit pip dependency.  I'm adding one for you, but still nagging you._

Reply:
I removed the dependency to `pip`, since there no pip installed modules left from the environment configuration file. But true, the recipe from the README does use pip.